apps/ledger-live-desktop/tests/fixtures/common.ts

```diff
@@ -101,6 +94,12 @@ const test = base.extend<TestFixtures>({
       timeout: 120000,
     });
 
+    await use(electronApp);
+
+    // close app
+    await electronApp.close();
+  },
+  page: async ({ electronApp }, use) => {
```

Comment:
I did that change at first because I needed to access `electronApp` inside the test but I can revert this change if you want